proto/cosmos/authz/v1beta1/query.proto

```diff
@@ -50,8 +55,24 @@ message QueryGranterGrantsRequest {
 
 // QueryGranterGrantsResponse is the response type for the Query/GranterGrants RPC method.
 message QueryGranterGrantsResponse {
-  // authorizations is a list of grants granted for grantee by granter.
-  repeated cosmos.authz.v1beta1.Grant grants = 1;
+  // grants is a list of grants granted by the granter.
+  repeated cosmos.authz.v1beta1.GrantAuthorization grants = 1;
```

Comment:
Is moving this to a different file within the same package actually a proto-breaking change?

Reply:
yeah I think that's fine. Also, IIRC you can just do:
```suggestion
  repeated GrantAuthorization grants = 1;
```

since it's all in the same package